libsession/src/main/java/org/session/libsession/utilities/recipients/RecipientProvider.java

```diff
@@ -104,7 +104,7 @@ boolean removeCached(@NonNull Address address) {
   }
 
   private @NonNull RecipientDetails getRecipientDetailsSync(Context context, @NonNull Address address, Optional<RecipientSettings> settings, Optional<GroupRecord> groupRecord, boolean nestedAsynchronous) {
-    if (address.isGroup()) return getGroupRecipientDetails(context, address, groupRecord, settings, nestedAsynchronous);
+    if (address.isGroupOrCommunity() && !address.isGroupV2()) return getGroupRecipientDetails(context, address, groupRecord, settings, nestedAsynchronous);
```

Comment:
Do we wan to exclude V2 here? If so wouldn't it make more sense to explicitly check for legacy OR community?

Reply:
It seems like the `getGroupRecipientDetails` is not loading anything for groupsv2. I think Harris might have made a decision not to pop in details into recipient for the new groups